quickstart/swarm/pom.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+     Copyright 2005-2015 Red Hat, Inc.
+
+     Red Hat licenses this file to you under the Apache License, version
+     2.0 (the "License"); you may not use this file except in compliance
+     with the License.  You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+     implied.  See the License for the specific language governing
+     permissions and limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.fabric8.quickstarts</groupId>
+        <artifactId>fabric8-quickstart-parent</artifactId>
+        <version>2.3-SNAPSHOT</version>
+    </parent>
+    <artifactId>swarm</artifactId>
+    <packaging>pom</packaging>
+    <name>Fabric8 :: Quickstarts :: Swarm :: Parent</name>
+
+    <modules>
+        <module></module>
```

Comment:
Submitted the missing module name

On Fri, Jul 1, 2016 at 3:46 PM, Jimmi Dyson <notifications@github.com>
wrote:

> In quickstart/swarm/pom.xml
> <https://github.com/fabric8io/ipaas-quickstarts/pull/1297#discussion_r69301429>
> :
>
> > +-->
> > +<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
> > +         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
> > +    <modelVersion>4.0.0</modelVersion>
> > +
> > +    <parent>
> > +        <groupId>io.fabric8.quickstarts</groupId>
> > +        <artifactId>fabric8-quickstart-parent</artifactId>
> > +        <version>2.3-SNAPSHOT</version>
> > +    </parent>
> > +    <artifactId>swarm</artifactId>
> > +    <packaging>pom</packaging>
> > +    <name>Fabric8 :: Quickstarts :: Swarm :: Parent</name>
> > +
> > +    <modules>
> > +        <module></module>
>
> At least it passed :-b
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/fabric8io/ipaas-quickstarts/pull/1297/files/eeb9b22a90617d25abe61e60dcef3abe3a9ca57c#r69301429>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAcTrplLpE8lbkQBWbflvem663HElSUSks5qRRo8gaJpZM4JDIlg>
> .
>



-- 
Charles Moulliard
Apache Committer & PMC / Architect @RedHat
Twitter : @cmoulliard | Blog :  http://cmoulliard.github.io